### DISCUSSION TITLE
rav1e.exe Trojan

### DISCUSSION THREAD

**Discussion by Zodler** [staxrip/staxrip#1037] [General]
StaxRip v2.22.0

Windows Defender

Trojan:Win32/Wacatac.B!ml
Alert level: Severe
Status: Active
Category: Trojan
Details: This program is dangerous and executes commands from an attacker

...Apps\Encoder\rav1e\rav1e.exe

**Comment by Dendraspis** [comment]
Possibly due to being UPX-packed, they often produce false positives. But if you wanna be 100% sure and save, there is always just one way to go: compile it by yourself, but that is also valid for StaxRip and its tools. 🤪
I have no suspicion that they are infected, I've tested some of them and had no issues so far.

**Comment by Zodler** [comment]
It just deleted it. What rav1e.exe is used for?

**Reply by Dendraspis** [reply]
It's an encoder. 

You could re-download StaxRip if you've deleted the archive.
But here is also the direct link:
https://www.mediafire.com/file/q17mp4ibrmk5g5z/rav1e-0.6.1-%2528p20230627-3-ge379128%2529-x64-gcc13.1.0.7z/file

**Reply by Zemciko** [reply]
Tried your link and result is same Windows defender marked it as infected with Wacatac.B!ml.
Same file in 2.21 archive. File from 2.20 archive looks ok or nothing is detected.

**Reply by Dendraspis** [reply]
Of course the result is the same, because it's the same file.

If you feel better with it, copy the one from `StaxRip v2.20` over and use that.

**Reply by Zodler** [reply]
I had major problems with my PC few hours later and had to do integrity check and windows found corrupt files. I don't know if it's related.

**Reply by Dendraspis** [reply]
Beside this report message, I don't have knowledge about any issues related to StaxRip, rav1e or so.

Like I said: You can use the old version of rav1e or compile it by your own or search the web for another binary. Nothing wrong with that. Chose a solution you feel confident with.
